geonode_mapstore_client/client/js/routes/Home.jsx

```diff
@@ -260,6 +260,8 @@ function Home({
             ...params,
             ...newParams
         }, pathname);
+
+        (isSmallDevice) ? handleShowFilterForm() : undefined;
```

Comment:
Please remove this statement and use explicit handler for filter apply and filter clear on the filter form. Add also some comment to explain the reason.